packages/@sanity/components/src/buttons/FalseButton.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import Ink from 'react-ink'
+import styles from 'part:@sanity/components/buttons/default-style'
+import Spinner from 'part:@sanity/components/loading/spinner'
+
+export default class DefaultButton extends React.Component {
```

Comment:
Class name does not match component name

Reply:
Agreed. Fake is a better name.